app/[lang]/error.tsx

```diff
@@ -40,7 +42,7 @@ export default function Error({
         </h2>
 
         <div className=" mt-8 w-full text-center font-cbrush text-2xl md:w-auto">
-          <Button href="/">{t('error.back_safety')}</Button>
+          <Button onClick={() => router.reload()}>{t('error.reload')}</Button>
```

Comment:
Looks like you were right about the router.reload() vs router.refresh() problem

though even after changing to router.refresh() i don't get the behavior i was expecting